### PR TITLE
:memo: New readme

### PR DESCRIPTION
**Pull request by ghost**
Added context and Installation and Run instructions

### CODE REVIEW

**Comment by moeenio** [comment]
@delnyn Some improvements. How do you like them?

TODO: About the content submodule.

**Comment by ghost** [comment]
Yes, thank you! But I think we should still say that the instructions in the readme only work on GNU/Linux.  
Also WDYM by "About the content submodule" ?

**Comment by moeenio** [comment]
> only work on GNU/Linux

Actually they don't (I think)

> Also WDYM by "About the content submodule" ?

The content of the website is in a seperate repository added as a submodule; it might be worth noting.

**Comment by ghost** [comment]
Okay

**Comment by ghost** [comment]
There you go @locness3

**Comment by moeenio** [comment]
*content in general not just posts

**Comment by ghost** [comment]
fuck I tagged the wrong person  
anyways, able to **squash and** merge @locness3?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
-<p align="center"><img src="logo-readme.png" alt="CatSource"></p>
+<img src="logo-readme.png" alt="CatSource"></p>
 
-_Tutorials, resources and more, for Scratchers, by Scratchers._
\ No newline at end of file
+CatSource is an open-source blog for tutorials about, resources and more for the [Scratch](https://scratch.mit.edu) coding program.  
+It is built as a static website on Jekyll (ruby), hosted on GitHub Pages.
+
+The al the content is in the [content submodule](https://github.com/csourcesc/posts).
```
Comment (moeenio) [inline README.md]: well no

Comment (ghost) [reply README.md]: uhm yeah I'm tired sorry

**Comment by ghost** [comment]
and now @locness3?

**Comment by ghost** [comment]
fuck I closed it by accident

**Comment by moeenio** [comment]
ily

**Pull request merged**